packages/vscode-extension/src/commandHandlers/params.ts

```diff
@@ -1,4 +1,5 @@
 import { DbSchema, lintCypherQuery } from '@neo4j-cypher/language-support';
+import { DEBOUNCE_TIME } from '@neo4j-cypher/react-codemirror/src/constants';
```

Comment:
We shouldn't be importing from `src` folders. And it's not worth it to add the entire `react-codemirror` package as a dependency so just duplicate the variable